cfgov/ask_cfpb/wagtail_hooks.py

```diff
@@ -132,10 +132,16 @@ def editor_css():
 
 
 def whitelister_element_rules():
```

Comment:
The placement of this hook in the `ask_cfpb` app and the title of this PR are a little misleading. These rules don't only apply to rich text fields in Ask -- they are global across the entire Wagtail site. See [the Wagtail docs](https://docs.wagtail.io/en/v1.13.4/reference/hooks.html#construct-whitelister-element-rules).

While you're modifying this, can we move this code into `v1.wagtail_hooks` to make it more obvious that it applies globally to the whole project?

Reply:
Cool, will do. Also realizing that I need to add in a bit of Less.

Reply:
Done in aa9fdf4.